source/arriving/setup_computer.rst

```diff
@@ -39,13 +278,19 @@ VPN
 SSH
     1. Connect to the University's VPN.
 
-    2. You must know your lab computer's IP address or its University code (ex: DINF-0000-00a). (Ask casius at https://casius.usherbrooke.ca/sp if you don't know).
+    2. You must know your lab computer's IP address or its University code (ex: DINF-0000-00a), which should be written on the computer. (Ask a research assistant if you don't know).
 
-    3. | Connect with ssh. On Linux or Mac, ssh can be simply used via the terminal. The option -X is to make sure the applications you use appear at home.
+    3. | Connect with ssh. On Linux, Mac or WSL (Windows), ssh can be simply used via the terminal. The option -X is to make sure the applications you use appear at home.
        | ``ssh -x your_cip@your_computer_IPaddress``, or
        | ``ssh -X your_cip@DINF-0000-00a.dinf.fsci.usherbrooke.ca``.
 
-       On Windows, you can use MobaXterm. Download it, then click on Session, SSH. In Remote host, enter your IP address. In Advanced SSH settings, make sure the X11-Forwarding button is clicked.
+       It can be useful to create an alias for this in your personal computer's .bashrc. Simply open the .bashrc and copy-paste this at the end of it (you can change the name ScilTour for whatever you want):
+
+       :bash:`alias ScilTour='ssh -X your_cip@DINF-0000-00a.dinf.fsci.usherbrooke.ca -o ServerAliveInterval=10'`
+
+    On Windows, you can also use MobaXterm. Download it, then click on Session, SSH. In Remote host, enter your IP address. In Advanced SSH settings, make sure the X11-Forwarding button is clicked.
```

Comment:
There are explanations here too that could be referenced
https://www.usherbrooke.ca/informatique/etudiants-actuels/faq/acces-a-distance-aux-serveurs-ubuntu